submissions/VolodymyrRutskyi/tiny-js-world/index.js

```diff
@@ -1,90 +1,52 @@
-/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
-   Complete the below for code reviewers' convenience:
+class Inhabitant {
+   constructor(species, name, gender, legs, saying){
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.legs = legs;
+      this.saying = saying;
+      this.props = ['species', 'name', 'gender', 'legs', 'saying']
+   }
+   showProps() {
+      return this.props.map(prop => this[prop]).join('; ');
+   }
+}
+
+class Human extends Inhabitant{
+   constructor (species, name, gender, saying){
+      super(species, name, gender, 2, saying);
+      this.hands = 2;
+      this.props = [...this.props, 'hands']
+   }
+}
+class Woman extends Human{
+   constructor(name, gender, saying){
+      super('woman', name, gender, saying)
```

Comment:
A woman can only be female, so there's no need to ask for a gender in the constructor here.